src/Akka.Streams.Kafka/Stages/Consumers/Abstract/SubSourceLogic.cs

```diff
@@ -331,25 +436,27 @@ private void HandleSubsourceCancelled((TopicPartition, ISubSourceCancellationStr
             }
         }
 
-        private void HandleSubsourceStarted((TopicPartition, IControl) obj)
+        private void HandleSubsourceStarted(SubSourceStageLogicControl sssLogicControl)
```

Comment:
Using the new data type here - this callback gets invoked once the substream starts